lib/resourcebuilder/apps.go

```diff
@@ -72,6 +76,34 @@ func (b *deploymentBuilder) Do(ctx context.Context) error {
 		}
 	}
 
+	// the CVO is the only operator which doesn't have knowledge of its own operand (itself).  If we detect the CVO deployment
+	// we need to replace the KUBERNETES_SERVICE_HOST env var with the internal load balancer to be resilient to kube-apiserver
+	// rollouts that cause the localhost server to become non-responsive for multiple minutes.
+	if deployment.Namespace == "openshift-cluster-version" && deployment.Name == "cluster-version-operator" {
+		infrastructureConfig, err := b.infrastructureGetter.Infrastructures().Get("cluster", metav1.GetOptions{})
+		// not found just means that we don't have proxy configuration, so we should tolerate and fill in empty
```

Comment:
how does "proxy configuration" come into this?  I'd expect that to be `Proxies().Get("cluster", ...)`

Reply:
> how does "proxy configuration" come into this? I'd expect that to be `Proxies().Get("cluster", ...)`

will fix comment

Reply:
f0f00ff9b9b has "we don't have infrastructure configuration yet".  Marking this thread resolved.